coops-ui/src/main.ts

```diff
@@ -35,6 +35,7 @@ configHelper.fetchConfig()
     // ensure we have a Keycloak token
     if (!sessionStorage.getItem('KEYCLOAK_TOKEN')) {
       console.log('Redirecting to Auth URL...')
+      sessionStorage.setItem('REDIRECTED', 'true')
```

Comment:
This is old code that was removed a while back. Did you restore it intentionally?

Reply:
no, I must have forgot to remove this, or I didn't see it in the rebase diff...my mistake